### PR TITLE
RE2022-175 Microtrait 2, initial Heatmap viz

### PR DESCRIPTION
**Pull request by dauglyon**
https://kbase-jira.atlassian.net/browse/RE2022-175

~Weirdly, this build runs fine locally. I also can't find the issue it is describing. So that's fun. @dakotablair could you try running the build on your system?~

Otherwise, this is ready for review.

### CODE REVIEW

**Comment by dauglyon** [comment]
Seems to be a combo of the following and whatever our stack is doing... https://stackoverflow.com/questions/70509055/postcss-preset-env-end-value-has-mixed-support-consider-using-flex-end-instead

File: src/features/collections/MatchPane.tsx [hunk 1]
```diff
@@ -34,7 +38,13 @@ const ViewMatch = () => {
     (state) => state.collections.currentSelection.length
   );
 
-  const matchQuery = usePollMatch(matchId);
+  const matchQuery = getMatch.useQuery(matchId || '', {
+    skip: !matchId,
+  });
+  useBackoffPolling(
+    matchQuery,
+    (result) => !!(result.error || result.data?.state !== 'processing')
```
Comment (dakotablair) [inline src/features/collections/MatchPane.tsx:46]: > Prefer Number/Boolean in new code

-- us, maybe, 2023-03-21

**Review by dakotablair — APPROVED** [review]

File: src/features/collections/data_products/HeatMap.module.scss [hunk 2]
```diff
@@ -0,0 +1,90 @@
+@import "../../../common/colors";
+
+.heatmap-wrapper {
+  display: grid;
+  font-size: 6px;
+
+  grid-template-columns: [left] auto [leftMargin] 1fr [right];
```
[the remark this answers is not in the document]
Comment (dauglyon) [reply src/features/collections/data_products/HeatMap.module.scss:7]: I believe Grid is actually impacting the layout performance so I may eliminate it.

File: src/features/collections/MatchPane.tsx (hunk 1 above)
Comment (dauglyon) [reply src/features/collections/MatchPane.tsx:46]: I'm going to add a "readable boolean logic" section to the europa meeting notes

**Pull request merged**